### ISSUE TITLE
Fine-tuning the TRN model (InceptionV3 backbone) trained on the Moments for something-something-v2 dataset

### ISSUE THREAD

**Issue by pencilzhang** [metalbubble/moments_models#16]
Hi Bolei:
Thanks for your great work!

I am using the TRN model trained on the Moments to fine-tune it on the sth-sth-V2 dataset.
The model seems to overfit after 25 epochs.  Right now I am using the default hyper parameters settings in TRN opts.

I am wondering if you did transferring TRN on the sth-sth dataset or any suggestions would be appreciate! Thanks!

**Comment by pencilzhang** [comment]
After observing many epochs, I found the above issue seems to be normal training loss fluctuation.

**Comment by pencilzhang** [comment]
still seems to be significant overfitting after 50 epochs...
